### DISCUSSION TITLE
Questions:

### DISCUSSION THREAD

**Discussion by shayanpartajli** [prosyslab-classroom/cs424-program-reasoning#407] [Q&A] [answered]
### Your Name

Shayan Partajli

### Question

Hello,

I have two questions:

Question 1:
I encountered an error when trying to run "make" in the root directory of the HW1 repository (please see the attached screenshot). Could someone please help me fix this error? Could this error be related to the warning mentioned in Question 2?

Question 2:
I followed the instructions to install the LLVM Primer (as explained here https://github.com/prosyslab-classroom/llvm-primer/blob/master/install.md). Clang/LLVM is installed on my system, and I have also updated Homebrew. However, when I tried to execute the command "brew install llvm@10.0.0", I received a warning (please see the attached screenshot). Could someone please advice me on how to resolve this and properly install LLVM version 10?

Best regards,
Shayan Partajli

Screenshot for Questions 1:
<img width="597" alt="Error" src="https://github.com/user-attachments/assets/401d863b-1400-4203-aa47-5271cac94406">

Screenshot for Questions 2:
<img width="578" alt="Warning" src="https://github.com/user-attachments/assets/90880f67-1a5e-45d9-a380-7214f1a77607">

**Comment by bongjunj** [comment, marked as answer]
I recommend you to use Docker or KCloud as this kind of dependency problems will appear again and again for all other assignments :)

**Comment by spearo2** [comment]
Thank you for your question.

As mentioned in the [instruction slide](https://docs.google.com/presentation/d/1rS39M1dui3khqTc6joWniNrxH3x87nGgOrWHiMAL4DU/edit#slide=id.gc3a8185f91_6_0) in the setup discussion post, It is not recommended to set it up on your own machine.

Is there a specific reason you chose to setup on your machine (e.g. KCloud is not working)?

If the problem continues on Docker or KCloud, please let us know.

**Comment by shayanpartajli** [comment]
Thank you for your fast response! I will try KCloud or Docker!
